Isaiah-Ogilvie-Submission/src/src code project.txt

```diff
@@ -0,0 +1,16 @@
+def is_prime(n):
+
+"""function defines a prime number."""
+
+	if n <= 1:
+		return f"{n} is not a prime number."
+	for i in range(2, int(n**0.5) + 1):
+		if n % i == 0:
+			return f"{n} is a complex number."
+		return f"{n} is a prime number."
```

Comment:
This return statement needs to be unindented! (otherwise it will tell you that 9 is a prime number!)
```suggestion
	return f"{n} is a prime number."
```